vulnerabilities/importers/debian_oval.py

```diff
@@ -52,35 +53,24 @@ def __init__(self, *args, **kwargs):
         # we could avoid setting translations, and have it
         # set by default in the OvalParser, but we don't yet know
         # whether all OVAL providers use the same format
-        self.translations = {'less than': '<'}
+        self.translations = {"less than": "<"}
         self.pkg_manager_api = DebianVersionAPI()
 
     def _fetch(self):
```

Comment:
Why marking it with _? This is a non trivial function that would need testing (and it does much more than just fetching as it is parsing  and returning parts of a PURL

Reply:
I've named methods with underscore prefix when they are intended to only be used internally(within only that file)  . 
Atleast that's what I interpret from https://pep8.org/#descriptive-naming-styles

> _single_leading_underscore: weak “internal use” indicator. E.g. from M import * does not import objects whose name starts with an underscore.

re. This is a non trivial function : It's basically downloading new files by leveraging the well tested `create_etag` function. That returning of parts of PURL is make the OvalDataSouce understand how to create PackageURL objects.   I'd call it borderline trivial :) . 

Anyways functions starting with `_`  doesn't mean they don't have tests or trivial, they are just too specialised to be used elsewhere.

Do you want to switch to plain function names ?